src/assets/stylesheets/components/_lazy-loader.scss

```diff
@@ -0,0 +1,103 @@
+.container-lazy-loader {
+  height: 100vh;
+  padding: rem(32px);
+
+  background: map-get($brand-black, 'light-1');
+}
+
+.lazy-loader-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__web-icon .react-loading-skeleton {
+    width: rem(117px);
+    height: rem(18px);
+
+    border-radius: rem(14px);
+  }
+
+  &__user-image .react-loading-skeleton {
+    width: rem(36px);
+    height: rem(36px);
+
+    border-radius: 50%;
+  }
+}
+
+.lazy-loader-body {
+  display: flex;
+  flex-direction: column;
+  max-width: rem(768px);
+  margin-top: rem(49px);
```

Comment:
This `margin-top` should be `40px`
![image](https://user-images.githubusercontent.com/14927672/123555875-32b3fa00-d7aa-11eb-869e-8bcb748ff815.png)

Reply:
Fixed in ac06f54f2beb361ad3d2b03ef845c09837f5e3b6